checker/tests/determinism/TestUpDet.java

```diff
@@ -32,20 +32,16 @@
         return retDetUp(list);
     }
 
-    public void testSubtypes(@PolyDet List<@Det String> l1, @PolyDet("down") List<@Det String> l2) {
+    public void testSubtypes(@PolyDet List<@Det String> l1, @PolyDet List<@Det String> l2) {
```

Comment:
This method and the next no longer test subtypes.  Do you have other methods that do so or should this be rewritten?